R/FilterEnsemble.R

```diff
@@ -0,0 +1,296 @@
+.FilterEnsembleRegister = new.env() # nolint
```

Comment:
Why do we need a registry for filter ensembles?

Reply:
We need to distinguish the ensemble filters from the normals ones to treat them more easily within the package, e.g. here

https://github.com/mlr-org/mlr/blob/e64fa48bb548bab6ec2e9eec611fd5ac5c020a2a/R/filterFeatures.R#L88

I just sticked to the "Register" approach to keep it consistent with the current implementations of simple filters.

Reply:
Hmm, a solution that uses properties like for the learners is probably better, but probably also too much work to implement.

Reply:
Yes, there's not really much time left in the box for this implementation here :sweat_smile: